packages/ui/VirtualList/VirtualList.less

```diff
@@ -2,6 +2,7 @@
 	position: relative;
 	height: 100%;
 	width: 100%;
+	overflow: hidden;
 
 	&.vertical {
 		overflow-x: hidden;
```

Comment:
Would you remove it because defining `overflow: hidden'?

Reply:
Good point. Updated.